ansible/files/multinode.sh

```diff
@@ -252,13 +257,15 @@ function build_kayobe_image() {
 
   # Set base image for kayobe container. Use rocky 9 by default
   export BASE_IMAGE=rockylinux:9
+  export USE_PYTHON_312=true
 
   if [[ "$(sudo docker image ls)" == *"kayobe"* ]]; then
     echo "Image already exists skipping docker build"
   else
     sudo DOCKER_BUILDKIT=1 docker build \
       --network host \
       --build-arg BASE_IMAGE=$BASE_IMAGE \
+      --build-arg USE_PYTHON_312=$USE_PYTHON_312 \
```

Comment:
Is this gonna break older deployments?

Maybe we could set USE_PYTHON_312 dynamically. We could read the content of .gitreview and check the value of  `defaultbranch` to get the openstack release

Reply:
How old are we talking? Caracal? Let's find out, I'll test this on Caracal.